webui/react/src/components/Navigation.tsx

```diff
@@ -39,10 +40,11 @@ const Navigation: React.FC<Props> = ({ children }) => {
     return () => canceler.abort();
   }, [canceler, fetchResourcePools]);
 
+  const makeListRoleCalls = useFeature().isOn('rbac');
```

Comment:
Would just call this `rbacEnabled` like we do elsewhere.

Reply:
✅